packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -249,6 +249,48 @@ describe('<Autocomplete />', () => {
       expect(getByRole('listbox')).to.have.property('scrollTop', 0);
     });
 
+    // https://github.com/mui/material-ui/issues/36212
+    it('should preserve scrollTop position of the listbox when adding new options on mobile', function test() {
```

Comment:
Please move the test outside of the parent describe block (`prop: autoHighlight`) since it is not related to it and keep it as a standalone test.

Reply:
Done 👍